R/algorithm.R

```diff
@@ -104,6 +116,18 @@ algorithm <- function() {
       logic = "diagnosekode =~ '^(DE11)'",
       comments = ""
     ),
+    lpr3_is_diabetes_code = list(
+      register = "diagnoser",
+      title = "LPR3 diagnoses codes for diabetes",
+      logic = "diagnosekode =~ '^(DE1[0-4])'",
```

Comment:
```suggestion
      logic = "diagnosekode =~ '^DE1[0-4]'",
```

The brackets aren't needed if there is no `|` involved.